test/e2e/density.go

```diff
@@ -395,20 +395,33 @@ var _ = framework.KubeDescribe("Density", func() {
 		{podsPerNode: 50, runLatencyTest: false, kind: api.Kind("ReplicationController")},
 		{podsPerNode: 95, runLatencyTest: true, kind: api.Kind("ReplicationController")},
 		{podsPerNode: 100, runLatencyTest: false, kind: api.Kind("ReplicationController")},
+		// Tests for other resource types:
```

Comment:
Yup. I agree - this is the reason why I originally wanted to move this to perf-test repo and create a separate binary that can read dedicated perf-test config or something. But I don't have time to do that...